tests/test_network.py

```diff
@@ -2,35 +2,115 @@
 from wfqc import network as nw
 import igraph
 import numpy as np
-import pickle
+import json
+import os
+import asyncio
+import shutil
+### Creating a test network to confirm the functions are working ###
 
-# def test_create_citation_network():
-#     G, included_tools = nw.create_citation_network(loadData=False, testSize=10) # TODO: if no data then make data or ask if make data?
-#     assert len (included_tools) > 0
+# Define the nodes 
+tools = ['TA', 'TB', 'TC', 'TD', # connected cluster - included in final graph 
+        # Separate cluster - included in final graph 
+         'TE', 'TF',
+        # Single disconnected cited - not included in final graph 
+         'TG', 
+        # Single disconnected not cited - not included in final graph 
+         'TH']
 
-def test_testsize_citation_network():
-    G, included_tools = nw.create_citation_network(loadData=False, testSize=10) # TODO: if no data then make data or ask if make data?
-    assert len (included_tools) > 0
+citations = ['CA', 'CB', 'CC', 'CD', 'CE', 'CF', 'CG', 'CH', 'CI', 'CJ', 'CK', 'CL', 'CM', 'CN', 'CO', 'CP', 'CQ']
 
-def test_load_citation_network():
-    G, included_tools = nw.create_citation_network(loadData=True) # TODO: if no data then make data or ask if make data?
-    assert len (included_tools) > 0
+
+edges = [
+
+    # Single citations of tools
+    ('CA', 'TA'), ('CB', 'TB'), ('CC', 'TC'), ('CD', 'TD'), 
```

Comment:
I would suggest creating a separate Python module named `example_network.py` that contains the example data and then use that data in this and tests/test_metrics.py test modules, for easier maintainability. You can then simplify the examples and call `Graph = igraph.Graph.TupleList(edges, directed=True)` and similar overlapping code directly in `example_network.py`.